apps/evm/src/constants/chainMetadata.ts

```diff
@@ -105,12 +114,24 @@ export const CHAIN_METADATA: {
     corePoolComptrollerContractAddress: '0xC527DE08E43aeFD759F7c0e6aE85433923064669',
     nativeToken: getToken({ chainId: ChainId.ZKSYNC_SEPOLIA, symbol: 'ETH' })!,
   },
-  [ChainId.ZKSYNC_MAINNET]: {
-    name: 'zkSync',
-    logoSrc: zkSyncLogo,
-    explorerUrl: 'https://explorer.zksync.io/',
+  [ChainId.OPTIMISM_MAINNET]: {
+    name: 'Optimism',
+    logoSrc: optimismLogo,
+    explorerUrl: 'https://optimistic.etherscan.io/',
     layerZeroScanUrl: 'https://layerzeroscan.com/',
-    corePoolComptrollerContractAddress: '0xddE4D098D9995B659724ae6d5E3FB9681Ac941B1',
-    nativeToken: getToken({ chainId: ChainId.ZKSYNC_MAINNET, symbol: 'ETH' })!,
+    blockTimeMs: 2000,
+    blocksPerDay: 43200,
+    corePoolComptrollerContractAddress: '', // TODO: add
+    nativeToken: getToken({ chainId: ChainId.OPTIMISM_MAINNET, symbol: 'ETH' })!,
+  },
+  [ChainId.OPTIMISM_SEPOLIA]: {
+    name: 'Optimism Sepolia',
+    logoSrc: optimismLogo,
+    explorerUrl: 'https://sepolia-optimism.etherscan.io/',
```

Comment:
I think this URL should not include the last `/`. There are other cases in this file.